src/main/scala/is/hail/distributedmatrix/GridPartitioner.scala

```diff
@@ -11,17 +11,27 @@ case class GridPartitioner(blockSize: Int, rows: Long, cols: Long, transposed: B
   val rowPartitions: Int = ((rows - 1) / blockSize + 1).toInt
   val colPartitions: Int = ((cols - 1) / blockSize + 1).toInt
 
+  def blockRowNRows(i: Int): Int =
```

Comment:
I call this `rowPartitionRows` in my write IRM as Block PR, but will change to this if you prefer it